README.md

```diff
@@ -22,90 +22,89 @@
 
 ---
 
-HyperFormula is a headless spreadsheet built on top of TypeScript. It is a parser and evaluator of Excel formulas for web applications. You can use it in a browser or as a service, with Node.js as your back-end technology.
-- High-speed Excel formula parsing and evaluating
-- A library of nearly [400 built-in functions](https://hyperformula.handsontable.com/guide/built-in-functions.html)
-- Support for [internationalization](https://hyperformula.handsontable.com/guide/i18n-features.html) with 17 built-in languages
-- Support for [custom functions](https://hyperformula.handsontable.com/guide/custom-functions.html)
+HyperFormula is a headless spreadsheet built in TypeScript, serving as both a parser and evaluator of spreadsheet formulas. It can be seamlessly integrated into your browser or utilized as a service with Node.js as your back-end technology.
+
+## What HyperFormula can be used for?
+HyperFormula doesn't assume any existing user interface, making it a great general-purpose library that can be used in various business applications. Here are some examples:
+
+- Custom spreadsheet-like app
+- Business logic builder
+- Forms and form builder
+- Educational app
+- Online calculator
+
+## Features
+
 - Function syntax compatible with [Microsoft Excel](https://hyperformula.handsontable.com/guide/compatibility-with-microsoft-excel.html) and [Google Sheets](https://hyperformula.handsontable.com/guide/compatibility-with-google-sheets.html)
+- High-speed parsing and evaluation of spreadsheet formulas
+- A library of [~400 built-in functions](https://hyperformula.handsontable.com/guide/built-in-functions.html)
+- Support for [custom functions](https://hyperformula.handsontable.com/guide/custom-functions.html)
 - [Support for Node.js](https://hyperformula.handsontable.com/guide/server-side-installation.html#install-with-npm-or-yarn)
```

Comment:
Let's make all the links on the list like this [Support for Node.js](https://hyperformula.handsontable.com/guide/server-side-installation.html#install-with-npm-or-yarn) - the entire list item should be wrapped in a link.